gapis/resolve/command_tree.go

```diff
@@ -546,3 +557,28 @@ func setRepresentations(ctx context.Context, g *api.CmdIDGroup, drawOrClearCmds
 		}
 	}
 }
+
+func getOpenGLAliasRepresentation(indices []uint64, item *api.CmdIDGroup, cmdTree *commandTree) api.CmdID {
+	aliasRepId := api.CmdNoID
+	if item.Name == "OpenGL ES Commands" {
+		parentIndices := indices[:len(indices)-1]
+		pItem, _ := cmdTree.index(parentIndices)
+		parentItem := pItem.(api.CmdIDGroup)
+		aliasRepId = parentItem.Range.Last()
+	} else if strings.HasPrefix(item.Name, "glDraw") || strings.HasPrefix(item.Name, "glMultiDraw") {
+		pindices := indices[:len(indices)-1]
+		pItem, _ := cmdTree.index(pindices)
+		switch parentItem := pItem.(type) {
```

Comment:
Since you only have one case statement, I think it's better to do:
```
if parentItem, ok := pItem.(api.CmdIDGroup); ok {
...
}
```
Same below.

Reply:
Thanks -- was looking for something like this!